src/main/resources/application.properties

```diff
@@ -27,11 +27,38 @@ iemr.session.expiry.time=1800
 enableIPValidation=false
 
 
+logging.pattern.level=INFO
 logging.level.root=INFO
-logging.level.org.springframework.web=INFO
-logging.level.org.hibernate=INFO
+logging.level.org.springframework.web=DEBUG
+logging.level.org.hibernate=DEBUG
 logging.level.com.iemr=DEBUG
-logging.level.org.springframework=INFO
+
+#### DB connection
+#spring.datasource.url=jdbc:mysql://10.208.122.31:3306/db_iemr
+#spring.datasource.username=piramaldev
+#spring.datasource.password=Piramal@2017
+#spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+#
+#spring.jpa.show-sql=true
+#spring.profiles.active=test
+#
+#common-url=http://10.208.122.31:8080/commonapi-v1.0/
 
 # slot size in mins multiple of 5
 scheduling-slotsize=5
+spring.datasource.url=jdbc:mysql://10.208.122.100:3306/db_iemr
```

Comment:
These details are not to be committed.
We had removed all Wipro IPs from the project.
It would help if you depended on the environment-level configuration.